### PR TITLE
FORUM-1377 : Birthday shown in the Public View profile

### PR DESCRIPTION
**Pull request by seifeddineDaoud**

### CODE REVIEW

File: component/core/src/main/java/org/exoplatform/social/core/identity/model/Profile.java [hunk 1]
```diff
@@ -49,6 +49,8 @@
   /** lastname key. */
   public static final String        FULL_NAME      = "fullName";
   
+  /** birthday key. */
+  public static final String        BIRTHDAY      = "birthday";
```
Comment (thomasdelhomenie) [inline component/core/src/main/java/org/exoplatform/social/core/identity/model/Profile.java]: Add blank line

File: component/core/src/main/java/org/exoplatform/social/core/identity/model/Profile.java [hunk 2]
```diff
@@ -219,7 +221,7 @@ public String getActivityId(ActivityProfileEntity entity) {
   private static final Map<UpdateType, String[]> updateTypes = new HashMap<UpdateType, String[]>();
   static {
     updateTypes.put(UpdateType.POSITION, new String[] {POSITION});
-    updateTypes.put(UpdateType.BASIC_INFOR, new String[] {FIRST_NAME, LAST_NAME, EMAIL});
+    updateTypes.put(UpdateType.BASIC_INFOR, new String[] {FIRST_NAME, LAST_NAME, EMAIL,BIRTHDAY});
```
Comment (thomasdelhomenie) [inline component/core/src/main/java/org/exoplatform/social/core/identity/model/Profile.java]: Is this change needed ? The updateTypes are the information of the Profile screen, and birthday is not in this screen.

Comment (seifeddineDaoud) [reply component/core/src/main/java/org/exoplatform/social/core/identity/model/Profile.java]: you're right, it's not necessary to add Birthday in updateTypes .
